test/test_ui.py

```diff
@@ -1238,63 +1238,69 @@ def tearDown(self):
         self.teardown_beets()
 
     def test_base(self):
-        l = self.run_with_output(u'ls')
-        self.assertEqual(l, u'the artist - the album - the title\n')
+        ltestbase = self.run_with_output(u'ls')
```

Comment:
Perhaps a better name for this variable would be `out` or something to indicate that it's the output log from the command.